tests.js

```diff
@@ -59,7 +71,7 @@ describe('through2-concurrent', function () {
       expect(transformCalls.length).to.be(6);
     });
 
-    it('should wait for all transform calls to finish before running flush', function () {
+    it('should wait for all transform calls to finish before running final and flush when processing > concurrency', function (callback) {
```

Comment:
The cases of processing > concurrency and processing < concurrency have been included because they include a drastically different code path:

* If `processing < concurrency`, callback of transform is called immediately
* Otherwise, callback is not called immediately